Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -116,19 +120,34 @@ public final class Environment: Environmenting {
         return !userOptedOut
     }
 
-    public var cacheDirectory: AbsolutePath {
+    public var cacheDirectory: Path.AbsolutePath {
```

Comment:
Why do we need the extra `Path.` when other references in this file don't need it?

Reply:
Odd. It caused a compilation issue, but it doesn't happen anymore if I remove it. I'll get rid of it.